docs/developer/tips-and-tricks.md

```diff
@@ -0,0 +1,41 @@
+# Tips and Tricks
+
+## Finding examples and documentation
+
+There are a number of examples you can try out on the https://arcade.makecode.com homepage.
+We also have several courses in development that are located [here](https://arcade.makecode.com/courses).
+API reference docs are located [here](https://arcade.makecode.com/reference).
+
+## "JavaScript" == "TypeScript"
+
+Despite the name shown in the MakeCode editor, our text language is actually a stricter subset of
+TypeScript. More information on what's supported can be found in the [language docs](https://makecode.com/language).
+
+## Source Control
+
+The MakeCode editor stores all of your projects in browser's local storage. The
+easiest way to back up a blocks project is to simply "download" it. For text
+projects, MakeCode has GitHub integration. Read more [here](https://makecode.com/extensions/github-authoring).
+
+MakeCode also supports "extensions" (libraries or packages) in projects, so it's possible to split your project
+into multiple repositories or libraries. Find more information [here](https://makecode.com/extensions).
+
+## Music and Art
+
+Documentation on how to import and use images in Arcade can be found [here](https://arcade.makecode.com/developer/images).
+
+Documentation on how to create music and sound effects is [here](https://arcade.makecode.com/developer/sound).
+
+## Getting help
+
+The public MakeCode discord is located at https://aka.ms/makecodecommunity and has a channel for arcade.
+
```

Comment:
To get help, you can ask the [MakeCode community](https://aka.ms/makecodecommunity) at/on our public discord channel for arcade.